src/data/cosmos.js

```diff
@@ -123,6 +123,14 @@ async function getFormDataByUserIdWithNewAuthToken(userId) {
   return item;
 }
 
+async function saveFormData(authToken, formData) {
+  const item = await getFormDataByAuthToken(authToken);
+  item.formData = formData;
+  item.confirmationNumber = uuidv4();
+  await insertForm(item);
```

Comment:
oh haha i made insertForm as a placeholder, so you can just replace this line with `await insertItem(item, formsContainerName);` and delete the insertForm function below

Reply:
Oh lol. Updated!